.gitignore

```diff
@@ -6,3 +6,5 @@ docker-compose_windows.yml
 
 #Webstorm folder
 **/.idea
+
+core/scripts/env
```

Comment:
We already have that in the `core/.gitignore` file

Reply:
addressed!